packages/alea-frontend/pages/api/nap/create-review-request.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import {
+  checkIfPostOrSetError,
+  executeAndEndSet500OnError,
+  getUserIdOrSetError,
+} from '../comment-utils';
+import { CreateReviewRequest } from '@stex-react/api';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfPostOrSetError(req, res)) return;
+  const userId = await getUserIdOrSetError(req, res);
+  const { reviewType, answerId } = req.body as CreateReviewRequest;
+  if (!reviewType || !answerId) return res.status(422).end();
+
+  await executeAndEndSet500OnError(
+    `INSERT INTO ReviewRequest (reviewType, answerId, userId) VALUES (?,?,?)`,
```

Comment:
Consider adding `questionId` here. Would be useful to have this information here.

Reply:
we discussed this on the meeting.